### PR TITLE
Warn when opening a truncated core file

### PR DESCRIPTION
**Pull request by keithc-ca**
With this, `jdmpview` will, like `jextract` warn about truncated core files:
```
$ jdmpview -core aix.core.dmp
DTFJView version 4.29.5, using DTFJ version 1.12.29003
Loading image from DTFJ...

Warning: dump file is truncated. Extracted information may be incomplete.
For a list of commands, type "help"; for how to use "help", type "help help"
Available contexts (* = currently selected context) : 

Source : file:///home/keithc/space/openj9/jdmpview/aix.core.dmp
	*0 : PID: 8323202 : No JRE

> 
```

### CODE REVIEW

File: jcl/src/openj9.dtfjview/share/classes/com/ibm/jvm/dtfjview/commands/OpenCommand.java [hunk 1]
```diff
@@ -128,100 +127,113 @@ private void imagesFromCommandLine(String args[]) {
...
-		if(!hasContexts) {
-			if(ctx.hasPropertyBeenSet(VERBOSE_MODE_PROPERTY)) {
-				out.println("Warning : no contexts were found, is this a valid core file ?");
+		if (!hasContexts) {
+			if (ctx.hasPropertyBeenSet(VERBOSE_MODE_PROPERTY)) {
+				out.println("Warning: no contexts were found, is this a valid core file?");
+			}
+		} else {
+			if (loadedImage.isTruncated()) {
+				out.println("Warning: dump file is truncated. Extracted information may be incomplete.");
```
Comment (pshipton) [inline jcl/src/openj9.dtfjview/share/classes/com/ibm/jvm/dtfjview/commands/OpenCommand.java:219]: I think there should be a blank line after this warning, to help it stand out from the help text.

**Comment by pshipton** [comment]
jenkins test sanity,extended aix jdk8

**Comment by pshipton** [comment]
PR build is https://ci.eclipse.org/openj9/job/PullRequest-OpenJ9/3243/

**Comment by pshipton** [comment]
Original PR build `Failed to connect to github.com port 443: Connection timed out`

jenkins test sanity,extended aix jdk8

**Comment by pshipton** [comment]
PR build is https://ci.eclipse.org/openj9/job/PullRequest-OpenJ9/3246

File: jcl/src/openj9.dtfjview/share/classes/com/ibm/jvm/dtfjview/commands/OpenCommand.java (hunk 1 above)
Comment (keithc-ca) [reply jcl/src/openj9.dtfjview/share/classes/com/ibm/jvm/dtfjview/commands/OpenCommand.java:219]: Agreed. I'll let the current build finish before pushing that change.

Comment (pshipton) [reply jcl/src/openj9.dtfjview/share/classes/com/ibm/jvm/dtfjview/commands/OpenCommand.java:219]: Sure. Doesn't matter though, the build will continue even if you push, and I've captured the link to make it easy to find.

**Comment by pshipton** [comment]
The PR builds passed.

File: jcl/src/openj9.dtfjview/share/classes/com/ibm/jvm/dtfjview/commands/OpenCommand.java (hunk 1 above)
Comment (keithc-ca) [reply jcl/src/openj9.dtfjview/share/classes/com/ibm/jvm/dtfjview/commands/OpenCommand.java:219]: Done.

**Comment by pshipton** [comment]
Reverting as this is causing a compile problem.
```
    [javac] Compiling 378 source files to /tmp/bld_443712/job_638887358/debugtools/DDR_VM/output/bin/j9_core
    [javac] /tmp/bld_443712/job_638887358/debugtools/DDR_VM/src/com/ibm/j9ddr/view/dtfj/image/J9DDRImage.java:336: error: method does not override or implement a method from a supertype
    [javac] 	@Override
    [javac] 	^
```

**Pull request merged**